encoding/raw/inbound.go

```diff
@@ -29,12 +29,13 @@ import (
 	"go.uber.org/yarpc/transport"
 )
 
-// rawHandler adapts a Handler into a transport.UnaryHandler
-type rawHandler struct {
-	UnaryHandler UnaryHandler
-}
+// rawUnaryHandler adapts a Handler into a transport.UnaryHandler
+type rawUnaryHandler struct{ UnaryHandler }
+
+// rawOnewayHandler adapts a Handler into a transport.OnewayHandler
+type rawOnewayHandler struct{ OnewayHandler OnewayHandler }
```

Comment:
You don't need to provide the field name if you're embedding like the other Handler.